modules/logging-loki-log-access.adoc

```diff
@@ -74,16 +74,36 @@ subjects:
 // tag::CustomAdmin[]
 == Custom admin group access
 
-If you have a large deployment with a number of users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. Admin users have access to all application logs in all namespaces, if they also get assigned the `cluster-logging-application-view` role.
+// tag::LokiMode[]
+If you have a large deployment with a number of users who require broader permissions, you can create a custom group using the `adminGroup` field.  Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. 
+// end::LokiMode[]
+
+// tag::NetObservMode[]
+If you need to see cluster-wide logs without being an administrator, you can create a custom group using the `adminGroup` field.  Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. 
```

Comment:
```suggestion
If you need to see cluster-wide logs without being an administrator, you can create a custom group using the `adminGroup` field.  Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR have the same read access to logs as admins. 
```

wdyt? It's to not make people think this would turn these users into admins

Reply:
I agree this is a good point to clarify, thanks for bringing it up 👍